MBTI Quiz/dist/style.css

```diff
@@ -42,157 +28,84 @@ li a:hover {
 	background-color: #111;
 }
 
-
 .navbar-custom {
-    background-color: #0D0B0B;
-	/*	background-color: #E33F3F; */
-}
-/* Main */
-
-.panel {
-	width: 60em;
-	margin-left: -20em
-}
-
-.card-title{
-	margin: 0;
-}
-
-.card-text{
-	margin: 0;
+	background-color: #0D0B0B;
 }
 
-.card-body{
-	padding: 0;
-}
+/*--------------------------------------------------------------
+# Main
+--------------------------------------------------------------*/
 
 .container-small {
-        width: 50%;
+	width: 50%;
 	padding-right: 15px;
-  padding-left: 15px;
-  margin-right: auto;
-  margin-left: auto;
-    }
-
-.center {
-	margin: auto;
-	width:20em;
-	text-align:center;
-}
-
-.preference{
-
-}
-
-.traits {
-
+	padding-left: 15px;
```

Comment:
Indenting on these css attributes is a bit extreme Standard is maybe 4 spaces

Reply:
fixed